irodsadmin/tests/test_commands.py

```diff
@@ -471,4 +500,34 @@ def test_command_multiple(self):
             self.project.full_title,
             orphan_path,
         )
-        self.assertEqual(expected, out.getvalue())
+        self.assertEqual(expected, output)
+
+    def test_command_ordering(self):
+        """Test ordering of orphans in command output"""
+        project1 = self.make_project('A_Project', PROJECT_TYPE_PROJECT, None)
+        self.make_assignment(project1, self.user, self.role_owner)
+        orphan_path1 = '{}/sample_data/study_{}'.format(
+            self.irods_backend.get_path(project1), str(uuid.uuid4())
+        )
+        # As the title of self.project is 'Test Project', it should be ordered
+        # after project1 with title 'A_Project'
+        orphan_path2 = '{}/sample_data/study_{}'.format(
+            self.irods_backend.get_path(self.project), str(uuid.uuid4())
+        )
+        self.irods.collections.create(orphan_path1)
```

Comment:
Generally we want to label multiple variables from the second one onwards, so the first one would just be `orphan_path`. This creates consistency and helps searching for the variables for when we e.g. only have a single such variable in other cases.

Reply:
But I've changed `orhpan_path1` to `orphan_path` and `orphan_path2` to `orphan_path1`. Could you check it one more time?

Reply:
It should be `orphan_path` and `orphan_path2`. Sorry if I was unclear earlier.